dev/customer_testing/run_tests.dart

```diff
@@ -0,0 +1,259 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:path/path.dart' as path;
+import 'package:glob/glob.dart';
+
+Future<void> main(List<String> arguments) async {
+  exit(await run(arguments) ? 0 : 1);
+}
+
+Future<bool> run(List<String> arguments) async {
+  final ArgParser argParser = ArgParser(
+    allowTrailingOptions: false,
+    usageLineLength: 72,
+  )
+    ..addOption(
+      'repeat',
+      defaultsTo: '1',
+      help: 'How many times to run each test. Set to a high value to look for flakes.',
+    )
+    ..addFlag(
+      'skip-on-fetch-failure',
+      defaultsTo: false,
+      help: 'Whether to skip tests that we fail to download.',
+    )
+    ..addFlag(
+      'skip-template',
+      defaultsTo: false,
+      help: 'Whether to skip tests named "template.dart".',
```

Comment:
You say "template.dart", but the code looks for "template.test"

Reply:
fixed